src/Featurizers/CountVectorizerFeaturizer.h

```diff
@@ -175,7 +23,7 @@ template <size_t MaxNumTrainingItemsV=std::numeric_limits<size_t>::max()>
 class CountVectorizerEstimator :
     public Components::PipelineExecutionEstimatorImpl<
         Components::DocumentStatisticsEstimator<MaxNumTrainingItemsV>,
-        Details::CountVectorizerEstimatorImpl<MaxNumTrainingItemsV>
+        Details::TfidfVectorizerEstimatorImpl<MaxNumTrainingItemsV>
```

Comment:
>Details::TfidfVectorizerEstimatorImpl [](start = 8, length = 37)

We shouldn't be reaching into the Details namespace for this functionality. Can the code contain a member instance of TfidfVectorizerEstimator?

Reply:
I'll reach out to discuss this as I am not sure how to do it